### ISSUE TITLE
[FEATURE REQ] Allow the option of `normal` verbosity for logging test runs during CI builds

### ISSUE THREAD

**Issue by jsquire** [Azure/azure-sdk-for-net#11730]
# Summary

When the NUnit framework is used for testing, it attempts to group test output written to the console or test context in order to ensure that the information for a given test and test fixture appear in logical order in the logs.   Because the ability exists to run tests in parallel, the framework accomplishes this by collecting all test output for a given fixture in memory, writing to the log only after the tests in that fixture have all completed.

Unfortunately, this means that using test log output for debugging is troublesome when a test hangs.  Because of the deferred nature of logging, if a test hangs the process is aborted and all cached output is lost.  Because multiple tests from different fixtures may be running in parallel, it is very difficult to understand what tests have run and which tests have not.

The `dotnet test` command offers the ability to tune the logging level and at its `normal` level, the test runner itself will print the outcome of a completed test as part of its progress.   This logging does not suffer from the caching issue of the NUnit framework and allows for a better understanding of which test have completed.   In the nighty test runs, `normal` verbosity is specified as part of the [archtype configuration](https://github.com/Azure/azure-sdk-for-net/blob/master/eng/pipelines/templates/jobs/archetype-sdk-tests.yml#L71).   However, in our CI builds, the logging verbosity is not specified in the [archtype configuration](https://github.com/Azure/azure-sdk-for-net/blob/master/eng/pipelines/templates/jobs/archetype-sdk-client.yml#L150) and defaults to a minimal state with no detail about individual tests.

# Desired Features:

- Allow for CI builds to use `normal` verbosity for test logging, preferably as the default,  or as an optional parameter.

# Related Pull Requests

- [[Engineering System] Enable Console Logging for CI Test Runs](https://github.com/Azure/azure-sdk-for-net/issues/11737) (#11737)

**Comment by jsquire** [comment]
I'd be happy to open a PR with the changes to the archtype, but am unsure if the verbosity level is intended as a redaction in the CI pipeline.   If the issues passes triage and you would prefer to that I make the change, please feel free to assign it back to me.
